sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -610,6 +623,49 @@ public void setSampleRate(Double sampleRate) {
     this.sampleRate = sampleRate;
   }
 
+  /**
+   * Returns the traces sample rate Default is null (disabled)
+   *
+   * @return the sample rate
+   */
+  public @Nullable Double getTracesSampleRate() {
+    return tracesSampleRate;
+  }
+
+  /**
+   * Sets the tracesSampleRate Can be anything between 0.01 and 1.0 or null (default), to disable
+   * it.
+   *
+   * @param sampleRate the sample rate
+   */
+  public void setTracesSampleRate(Double sampleRate) {
+    if (sampleRate != null && (sampleRate > 1.0 || sampleRate <= 0.0)) {
```

Comment:
this `if` is pretty much the same as `setSampleRate`, what's about extracting to a private method and taking the `sampleRate` as a param?

Reply:
👍